### PR TITLE
docs: cleanup

### PR DESCRIPTION
**Pull request by Steven-Gassert**
#### What do these changes do/fix?
This cleans up some of the information in our docs

### CODE REVIEW

**Comment by Steven-Gassert** [comment]
i figured we no longer need `docs/getting-started/contributing.md ` now that we have `.github/CONTRIBUTING.md`

File: docs/_config.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-title: Discovery Widgets Docs
-description: Discovery Widgets Package
+title: Discovery Components Docs
+description: Discovery Components
 remote_theme: pmarsceill/just-the-docs
 search_enabled: true
-repo_code_url: "https://github.ibm.com/watson-discovery/disco-widgets/blob/master/"
+repo_code_url: "https://github.com/watson-developer-cloud/discovery-components"
 yo_dawg: "![](https://imgflip.com/s/meme/Yo-Dawg-Heard-You.jpg)"
```
Comment (maniax89) [inline docs/_config.yml]: remove this

File: docs/index.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ title: Home
 nav_order: 0
 ---
 
-# Discovery Widget Docs
+# Discovery Components Docs
 
-With Discovery Widgets you can do all sorts of mystical and magical stuff. Like fine unicorns, slay dragons or fit more than 44 marshmellows in your mouth at one time.
```
Comment (maniax89) [inline docs/index.md]: remove this

File: docs/index.md [hunk 3]
```diff
@@ -4,8 +4,8 @@ title: Home
...
+With Discovery Components you can do all sorts of mystical and magical stuff. Like find unicorns, slay dragons or fit more than 44 marshmellows in your mouth at one time.
```
Comment (maniax89) [inline docs/index.md]: remove this

**Pull request merged**